### ISSUE TITLE
Scalp erythematous plaque

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11582]
**Preferred term label:**

**Synonyms**
Erythematous plaque of the scalp

**Definition (free text, please give PubMed ID)**
PMID: 42074677 Characteristic skin findings include heliotrope rash, shawl sign, V-sign, nailfold abnormalities (Keining’s sign), and **scalp dermatitis**.

PMID: 31279808 Scalp involvement: Atrophic, erythematous, sometimes pruritic scaly plaques. May be misdiagnosed as psoriasis or seborrheic dermatitis.

PMID: 37153943 Scalp can be also involved, showing a dusky erythematous itchy scaly dermatitis (figure 3a), which is usually misdiagnosed as seborrheic dermatitis or psoriasis. In some patients, a non-scarring alopecia has been reported, usually in association with a flare of DM. figure 3: Scalp involvement in dermatomyositis. a) erythematous plaques of the scalp

**Parent term (use hpo.jax.org/app)**
Skin plaque  HP:0200035

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dermatomyositis ORPHA:221

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Label: Erythematous scalp plaque
Def: A circumscribed, palpable, potentially elevated, erythematous lesion > 1 cm in diameter and localized to the scalp. Erythematous plaques exhibiting erythema, i.e., abnormal redness of the skin due to localized dilation and congestion of superficial dermal capillaries, typically as a result of inflammation, irritation, infection, or vasodilation, and usually blanching with pressure.
Comment: Erythematous plaque of the scalp may be observed in dermatomyositis and often are pruritic.
PMID:37153943
PMID:26801523

TODO also check related definitions with PMID: 26801523
